popup.js

```diff
@@ -39,8 +41,8 @@ function onLoad() {
   document.getElementById("textTransform").addEventListener("change", onOptionChange);
   document.getElementById("textDecoration").addEventListener("change", onOptionChange);
   document.getElementById("fontSize").addEventListener("change", onOptionChange);
-  document.getElementById("fonts-subset").addEventListener("change", onOptionChange);
-  document.getElementById("fonts-searchbox").addEventListener("input", onSearchChange);
+  document.getElementById("fonts-subset").addEventListener("change", (() => debounce(onSubsetChange))());
+  document.getElementById("fonts-searchbox").addEventListener("input", (() => debounce(onSearchChange))());
```

Comment:
I don't think there's a need to use an IIFE here, right?
(IIFE: https://developer.mozilla.org/en-US/docs/Glossary/IIFE)
I think it can just be:
  document.getElementById("fonts-subset").addEventListener("change", debounce(onSubsetChange));
  document.getElementById("fonts-searchbox").addEventListener("input", debounce(onSearchChange));